### PR TITLE
added budget, entity, exemption, procurement and support api.

### PR DESCRIPTION
**Pull request by ronmamo**
Also switched to sqlalchemy (db agnostic) and restplus (swagger+).
Migrated few apis.

### CODE REVIEW

File: open_budget_data_api/models.py [hunk 1]
```diff
@@ -1,52 +1,196 @@
...
+    personnel_revised = db.Column(db.Float)
+    commitment_allocated = db.Column(db.Integer)
+    commitment_revised = db.Column(db.Integer)
+    amounts_allocated = db.Column(db.Integer)
+    amounts_revised = db.Column(db.Integer)
+    contractors_allocated = db.Column(db.Integer)
+    contractors_revised = db.Column(db.Integer)
+    dedicated_allocated = db.Column(db.Integer)
+    dedicated_revised = db.Column(db.Integer)
+    # equiv_code = ARRAY(db.String)
```
Comment (akariv) [inline open_budget_data_api/models.py]: Why did you remove those arrays?

**Comment by mushon** [comment]
If I did it was not deliberately. Please revert these changes.

File: open_budget_data_api/models.py (hunk 1 above)
Comment (ronmamo) [reply open_budget_data_api/models.py]: Fixed. Thanks.
(I needed to move fast then and arrays didn't work ootb for me. I've continued and refined this work in the second branch (and PR) - graphql.)

**Comment by akariv** [comment]
Closed via #3
